spec.bs

```diff
@@ -7284,6 +7330,31 @@ for a given [=k-anonymity key=]. These records are stored in the [=user agent=].
   :: The [=moment=] when the k-anonymity status in this record was last fetched.
 </dl>
 
+<h3 id=auction-data-config-header>auction data config</h3>
+
+An <dfn>auction data config</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="auction data config">
+  :  <dfn>seller</dfn>
+  :: An [=origin=].
+    The origin of the seller running the ad auction. The [=origin/scheme=] must be "`https`".
+  :  <dfn>coordinator</dfn>
+  :: The origin of the coordinator hosting public encryption keys for the server
+    running the ad auction. The [=origin/scheme=] must be "`https`".
+  :  <dfn>request size</dfn>
+  :: {{unsigned long}} or null. An optional field, containing the desired size
+    for the returned encrypted request blob.
+  :  <dfn>per buyer config</dfn>
+  :: A [=map=] whose [=map/keys] are [=origins=] and [=map/values=] are [=auction data buyer config=].

```

Comment:
Missing trailing = on map/keys here.
(Noticed while looking for inspiration for things to WPT)

Reply:
Fixed.